packages/flutter_tools/lib/src/commands/build_apk.dart

```diff
@@ -155,6 +156,9 @@ class BuildApkCommand extends FlutterCommand {
     argParser.addOption('flx',
       abbr: 'f',
       help: 'Path to the FLX file. If this is not provided, an FLX will be built.');
+    argParser.addOption('add-file',
+      help: 'Add a file to the APK.  Must have the format [path in APK]=[local file path]',
```

Comment:
Perhaps: `Add a file to the APK (must have the format <path/in/APK>=<local/file/path>).`.

Whatever text you think works best for the cli option; we should use a full sentence through, to match the other options.